### DISCUSSION TITLE
Existing AFM/SPM analysis software

### DISCUSSION THREAD

**Discussion by derollins** [AFM-SPM/home#9] [General]
It is always helpful to compile lists of software that is already out there and available. Here is Pete Eatons: https://www.afmhelp.com/index.php?option=com_content&view=article&id=47&Itemid=53 

A software list is also being compiled here: https://github.com/AFM-SPM/home/blob/main/software.csv

**Comment by paulmueller** [comment]
I have created a repository for the software list: https://github.com/AFM-SPM/afm-software-list

Everything should be quite self-explanatory. If you would like to add an entry, you can do so via an issue or via a pull request.

The workflow is completely automated. New entries automatically show up here: https://afm-spm.github.io/afm-software-list/

All issues related to the software list should be discussed here: https://github.com/AFM-SPM/afm-software-list/issues

Please add new entries and/or update the current entries :heart:!

@derollins Please remove the software.csv file in the home repository and link to  https://afm-spm.github.io/afm-software-list/ everywhere it makes sense.

Cheers,
Paul
